pkg/common/constants.go

```diff
@@ -33,23 +35,28 @@ const (
 	// DatstorePath
 	DatstorePath = "data-stores"
 	// Virtual-Images Path
-	ResourcePoolPath       = "resource-pools"
-	NetworkRouterPath      = "routers"
-	NetworkRouterTypePath  = "network-router-types"
-	NetworkServicePath     = "services"
-	NetworkScopePath       = "scopes"
-	ServerPath             = "servers"
-	VirtualImagePath       = "virtual-images"
-	FolderPath             = "folders"
-	ConfigOptionPath       = "config-options"
-	PowerSchedulPath       = "power-schedules"
-	EnvironmentPath        = "environments"
-	OptionsPath            = "options"
-	ZoneNetworkOptionsPath = "zoneNetworkOptions"
-	ProvisionTypesPath     = "provision-types"
-	RouterRoutePath        = "routes"
-	RouterBgpNeighborPath  = "bgp-neighbors"
-	NetworkEdgeClusterPath = "edge-clusters"
+	ResourcePoolPath               = "resource-pools"
+	NetworkRouterPath              = "routers"
+	LoadBalancerPath               = "load-balancers"
+	LoadBalancerMonitorPath        = "monitors"
+	LoadBalancerProfilePath        = "profiles"
+	LoadBalancerPoolPath           = "pools"
+	LoadBalancerVirtualServersPath = "virtual-servers"
+	NetworkRouterTypePath          = "network-router-types"
+	NetworkServicePath             = "services"
+	NetworkScopePath               = "scopes"
+	ServerPath                     = "servers"
+	VirtualImagePath               = "virtual-images"
+	FolderPath                     = "folders"
+	ConfigOptionPath               = "config-options"
+	PowerSchedulPath               = "power-schedules"
+	EnvironmentPath                = "environments"
+	OptionsPath                    = "options"
+	ZoneNetworkOptionsPath         = "zoneNetworkOptions"
+	ProvisionTypesPath             = "provision-types"
+	RouterRoutePath                = "routes"
+	RouterBgpNeighborPath          = "bgp-neighbors"
+	NetworkEdgeClusterPath         = "edge-clusters"
 
 	WhoamiPath = "whoami"
```

Comment:
to maintain consistency over the convention in the file, please add a comment over whoami path